modules/ocf_desktop/files/xsession/lightdm/lightdm-gtk-greeter.po

```diff
@@ -0,0 +1,3 @@
+#: ../src/lightdm-gtk-greeter.c:1479
```

Comment:
I don't think the references are actually significant except to humans?
http://pology.nedohodnik.net/doc/user/en_US/ch-poformat.html